src/app/components/list/view-switcher/index.html

```diff
@@ -21,44 +21,44 @@
       <h3 class="sky-text-success">Use when</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-              <p>Use the list view switcher when the user needs to switch between different views of the same data.</p>
+              <p>Use list view switchers when users need to switch between different views of the same data.</p>
           </sky-docs-visual-example-text>
           <sky-docs-visual-example-image>
             <div>
               <img src="~/assets/img/guidelines/list-view-switcher/use-when.png" />
-              <div class="caption caption-success sky-text-success">Do use the list view switcher when there is more than one view of the data.</div>
+              <div class="caption caption-success sky-text-success">Do use list view switchers when more than one view of data is available to users.</div>
             </div>
           </sky-docs-visual-example-image>
         </sky-docs-visual-example>
-      <h3 class="sky-text-success">Don't use when</h3>
+      <h3 class="sky-text-danger">Don't use when</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-              <p>Don't use the list view switcher when there is only one view of the data.</p>
+              <p>Don't use list view switchers when only one view of data is available.</p>
           </sky-docs-visual-example-text>
           <sky-docs-visual-example-image>
             <div>
               <img src="~/assets/img/guidelines/list-view-switcher/dont-use-when.png" />
-              <div class="caption caption-danger sky-text-danger">Don't use the list view switcher when there is only one view of the data.</div>
+              <div class="caption caption-danger sky-text-danger">Don't use list view switchers when users cannot switch between different views.</div>
             </div>
           </sky-docs-visual-example-image>
         </sky-docs-visual-example>
     </sky-demo-page-content>
 
     <sky-demo-page-content sectionHeading="Anatomy">
       <sky-docs-anatomy imagePath="~/assets/img/guidelines/list-view-switcher/anatomy.png">
-        <sky-docs-anatomy-item anatomyNumber="1">Active view</sky-docs-anatomy-item>
-        <sky-docs-anatomy-item anatomyNumber="2">Inactive view</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="1">Active view button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="2">Inactive view button</sky-docs-anatomy-item>
       </sky-docs-anatomy>
     </sky-demo-page-content>
 
     <sky-demo-page-content sectionHeading="Options">
       <h3>Available views</h3>
         <div class="sky-text-block">
-          <p>You must specify which views are available for the user to switch between.</p>
+          <p>You must specify the views that are available to users.</p>
```

Comment:
@Blackbaud-AlexKingman, how do users specify the views? Looking in the code, the only place where I see to specify anything is the custom button component. Is that where this is specified? And if so, how is this different than the custom views we mention two subsections down?

Reply:
I think the issue right now is that there is only one built in mode to choose from

Reply:
So this is where things get tricky. Today, we do not document how to create custom views. I do know we have a couple consumers that dug into the code and are doing it but we've never documented it as I don't know how main stream we wanted to make it. The design documentation for the view switcher specifically called out supporting custom views though and so it does just that. I feel like documenting the view switcher at all isn't extremely helpful today if we don't also document the custom views (since there is only one supported view today) but I don't know what @Blackbaud-ToddRoberts thinks.

Reply:
I think we need to document creating custom views. There is nothing inherently wrong with using them if we don't have a suitable built in view.

Reply:
@Blackbaud-ToddRoberts and @Blackbaud-TrevorBurch, where would we document custom views? I assume on the list docs as opposed to here. Is that right? If so, I think we need to create a separate issue to document custom views. Please let me know what you think.

Reply:
I'm not sure the best place. Will the method be changing with the new data manager?

Reply:
I would agree that the custom view work probably fits better on the main list topic. And as for the new data manager - this component is only for the old one. I'm not sure how we will want to resolve/replace current list stuff with that when it is available but I don't think we should attempt to solve that problem with this component.

Reply:
Great. I'm resolving this issue here and creating a separate docs issue at https://github.com/blackbaud/skyux2-docs/issues/825.